website/docs/guides/migration/versions/04-upgrading-to-v1.4.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Upgrading to v1.4 (latest)"
+description: New features and breaking changes for dbt Core v1.4
```

Comment:
Suggest remove "breaking changes," just given the note below. There shouldn't be any breaking changes, but some changes that might require some action/awareness upon upgrading, in ways that we've communicated + set expectations around :)
```suggestion
description: New features and changes in dbt Core v1.4
```